css-brochure-form/css-brochure-form/styles.css

```diff
@@ -0,0 +1,57 @@
+body {
+  margin: 100px;
+  max-width: 500px;
+  max-height: 500px;
+}
```

Comment:
I don't know why you are doing this, but it can be removed. As a rule, you should never limit the size of the `body` element.